modules/ztp-generating-install-and-config-crs-manually.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-far-edge/ztp-manual-install.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-generating-install-and-config-crs-manually_{context}"]
+= Generating ZTP installation and configuration CRs manually
+
+You can generate zero touch provisioning (ZTP) site installation and configuration custom resources (CRs) manually by using the `ztp-site-generate` container. Do this when you want to manually install a managed cluster.
+
+.Prerequisites
+
+* You have configured the hub cluster for generating the required installation and policy custom resources (CRs).
+
+.Procedure
+
+. Export the `argocd` directory from the `ztp-site-generate` container image using the following commands:
++
+[source,terminal]
+----
+$ podman pull registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.10
+----
++
+[source,terminal]
+----
+$ mkdir -p ./out
+----
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run --log-driver=none --rm registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version} extract /home/ztp --tar | tar x -C ./out
```

Comment:
This cmd does not work. Please check with SME on the expected command. 
```
[kni@registry.kni-qe-15 ~]$ podman run --log-driver=none --rm registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.11 Build extract /home/ztp --tar | tar x -C ./out
Error: container_linux.go:380: starting container process caused: exec: "Build": executable file not found in $PATH: OCI not found
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors

```

Reply:
updated